leanpkg.toml

```
@@ -1,7 +1,7 @@
 [package]
 name = "mathlib"
 version = "0.1"
-lean_version = "leanprover-community/lean:3.32.1"
+lean_version = "urkud/lean:LOFIX"
```

```suggestion
lean_version = "leanprover-community/lean:3.33.0"
```

still compiling on CI, but nearly there!

---

The Linux binary just appeared so let's give it a shot.